ir/program/function_signature.rs

```diff
@@ -26,7 +26,7 @@ pub enum FunctionID {
 
 pub struct FunctionSignature {
     pub(crate) function_id: FunctionID,
-    pub(crate) arguments: Vec<(VariableCategory, VariableOptionality)>,
+    pub(crate) arguments: Vec<(VariableCategory, VariableOptionality)>, // TODO: Arguments cannot be optional
```

Comment:
Can they?

Reply:
Not at the moment no! But let's cross check w Christoph

Reply:
Then why do we have the kwargs style syntax? Just to make things explicit?

Reply:
where do we have that?

Reply:
Idk, I just remember them being suggested.